packages/about-you/theme/nuxt.config.js

```diff
@@ -43,10 +43,12 @@ export default {
         ]
       }
     }],
+    // @core-development-only-start
```

Comment:
**missing docs:** Please include a paragraph about this magic comment in theme docs